R/endpoints.R

```diff
@@ -64,9 +64,9 @@ endpoint_model_options <- function(options_template) {
 }
 
 endpoint_model_submit <- function(queue) {
-  function(req, res, data, parameters) {
+  function(req, res, data, options, advanced) {
```

Comment:
Ooh, unless..... we just group these all together into a single "options" dictionary. Would make the interface simpler. @r-ash any thoughts on whether model options and advanced options should be distinguished in this way?

Reply:
I don't mind - it will be fairly straight forward to use those if they are together or separate.

I agree makes the interface a little simpler but either is fine from my point of view.

Reply:
I think it's more flexible to group them actually. That way if, say we didn't end up having any advanced options, or if we ended up having 3 sets of options, the schema/interface remains unchanged.

Reply:
Yeah great point - let's group them

Reply:
Ok, have updated accordingly.